src/components/helpers/dom_helpers.ts

```diff
@@ -14,3 +14,7 @@ export function gridOverlayPosition() {
   }
   throw new Error("Can't find spreadsheet position");
 }
+
+export function getOpenedMenus(): HTMLElement[] {
+  return Array.from(document.querySelectorAll<HTMLElement>(".o-menu"));
```

Comment:
```suggestion
  return Array.from(document.querySelectorAll<HTMLElement>(".o-spreadsheet .o-menu"));
```
Just to be sure to avoid conflict with the rest of the world